components/case-studies-card/case-studies-card.module.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 2.25rem;
+
+  @include breakpoint(small) {
+    margin: 0;
+    max-width: 100%;
+    width: 100%;
+  }
+
+  @include breakpoint(large) {
+    max-width: 34.875rem;
+  }
+}
+
+.logoContainer {
+  width: 100%;
+  height: 21.4375rem;
+  background-color: $color-dark;
+  border: 1px solid $color-grey-border;
+  border-radius: 0.8125rem;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  position: relative;
+
+  @include breakpoint(small) {
+    width: 100%;
+    height: 13.1875rem;
+    border-radius: 0.5rem;
+  }
+
+  @include breakpoint(medium-down) {
+    width: 100%;
+    border-radius: 8px;
+    height: 21.4375rem;
+  }
+
+  @include breakpoint(large) {
+    max-width: 34.875rem;
+  }
+}
+
+.textContainer {
+  padding: 66px 0;
```

Comment:
The spacing still looks too large here. I think this should be `padding: 34px 0 66px;` and that's because the `h2` has both margin and padding on it, which extends the space even more than 66px.

<img width="523" alt="image" src="https://user-images.githubusercontent.com/16869061/157490100-1ec64e86-f94e-4eb2-830c-4dfbc882932a.png">